static/app/components/events/eventReplay/replayContent.spec.tsx

```diff
@@ -30,8 +30,8 @@ const mockedReplay: Partial<ReplayReader> = {
     dist: '',
     traceIds: [],
     errorIds: ['5c83aaccfffb4a708ae893bad9be3a1c'],
-    startedAt: new Date(),
-    finishedAt: new Date(),
+    startedAt: new Date(1663025353247.3),
```

Comment:
lets do something that's easy to read like:

```
const mockStartedAt = 'Sep 12, 2022 11:29:13 PM UTC';
const mockFinishedAt = 'Sep 15, 2022 17:22:07 PM UTC';

...
startedAt: new Date(mockStartedAt),
finishedAt: new Date(mockFinishedAt)
...

expect(...)..toHaveTextContent(mockStartedAt);
```

Then i don't have to manually convert milliseconds to know what the test is doing.